From cc910c4b180f3adc955ddb01e7c2948a82fdd815 Mon Sep 17 00:00:00 2001
From: Jamie Lennox <jamielennox@redhat.com>
Date: Mon, 23 Sep 2013 16:11:53 +1000
Subject: [PATCH] Make loading description python independent

README.rst contains some unicode characters which under certain
environment conditions will cause python3 to fail to read the
description and run setup.py.

fixes: #186
---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 2cb1555..493d85d 100644
--- a/setup.py
+++ b/setup.py
@@ -1,4 +1,5 @@
 import imp
+import io
 import sys
 from os import path
 from setuptools import setup, find_packages, Extension
@@ -53,7 +54,7 @@ setup(
     name='falcon',
     version=VERSION,
     description='A supersonic micro-framework for building cloud APIs.',
-    long_description=open('README.rst', 'r').read(),
+    long_description=io.open('README.rst', 'r', encoding='utf-8').read(),
     classifiers=[
         'Development Status :: 4 - Beta',
         'Environment :: Web Environment',
-- 
1.8.3.1

